helpers/navigation.js

```diff
@@ -4,6 +4,10 @@ const Helper = require('codeceptjs/lib/helper');
 
 module.exports = class Navigation extends Helper {
   refreshPage() {
-    this.helpers.WebDriverIO.browser.refresh();
+    return new Promise((resolve, reject) => {
+      this.helpers.WebDriverIO.browser.refresh()
```

Comment:
Is this necessary, couldn't you just `return this.helpers.WebDriverIO.browser.refresh()`, which would return the thenable to the caller?